index.js

```diff
@@ -1,31 +1,15 @@
-import {promisify} from 'node:util';
 import path from 'node:path';
 import process from 'node:process';
 import {globby, globbySync} from 'globby';
 import isGlob from 'is-glob';
-import slash from 'slash';
-import gracefulFs from 'graceful-fs';
 import isPathCwd from 'is-path-cwd';
 import isPathInside from 'is-path-inside';
-import rimraf from 'rimraf';
 import pMap from 'p-map';
-
-const rimrafP = promisify(rimraf);
+import {rimraf} from 'rimraf';
```

Comment:
I don’t think this is needed anymore, right? This has been available since node 14.14

``` js
fs.rmSync(dir, { recursive: true, force: true });
``` 

Globing, if needed, could still be supported directly, via fdir: https://www.npmjs.com/package/fdir

Reply:
Do you mean to say `rimraf` as a whole isn't needed in `del`?